src/JsonApiDotNetCore.MongoDb/JsonApiDotNetCore.MongoDb.csproj

```diff
@@ -1,12 +1,29 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <VersionPrefix>4.0.0-beta1</VersionPrefix>
+    <TargetFramework>$(NetCoreAppVersion)</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <PackageTags>jsonapi;json:api;dotnet;core;mongodb</PackageTags>
+    <Description>Persistence layer implementation for use of mongodb in applications using JsonApiDotNetCore</Description>
```

Comment:
'MongoDB' seems to be the official casing, according to https://www.mongodb.com/